download.py

```diff
@@ -12,6 +12,7 @@
 
 JPL = "ftp://ssd.jpl.nasa.gov/pub/eph/planets/bsp"
 IERS = "ftp://ftp.iers.org/products/eop/rapid/standard"
+IERS_WEB = "https://datacenter.iers.org/data/9"
```

Comment:
The url comes from astropy
https://github.com/astropy/astropy/blob/main/astropy/utils/iers/iers.py#L66